### PR TITLE
AstSetSystemProperty: Enables developers modify behavior of running H2Os

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

**Comment by Pscheidl** [comment]
Junit8 failure reason:

```plain
12:09:30 [h2o-3-pipeline_PR-2324-FTTYHKYAJQ5ERF35JGUGTWJGOURLDF5JKZRJ37NOIU2GYX77RAYQ] Running shell script
12:09:30 Error response from daemon: Container 2963f81e39ec8dfaa924283829fc014a765040717f738bafa7f18db50c26e470 is not running
12:14:37 process apparently never started in /home/jenkins/slave_dir_from_mr-0xb1/workspace/h2o-3-pipeline_PR-2324-FTTYHKYAJQ5ERF35JGUGTWJGOURLDF5JKZRJ37NOIU2GYX77RAYQ@tmp/durable-e98ada04
```

Re-triggered built (failed only).

File: h2o-core/src/test/java/water/rapids/ast/prims/misc/SetClusterPropertyTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+public class SetClusterPropertyTaskTest extends TestUtil {
+
+  @BeforeClass
+  public static void setup() {
+    stall_till_cloudsize(5);
+  }
+
+  @Test
+  public void testSetClusterProperty() {
```
Comment (Pscheidl) [inline h2o-core/src/test/java/water/rapids/ast/prims/misc/SetClusterPropertyTaskTest.java]: As this is a Junit test, the property being properly set in the environment can be tested as well by System.getEnv()

Comment (michalkurka) [reply h2o-core/src/test/java/water/rapids/ast/prims/misc/SetClusterPropertyTaskTest.java]: It is tested on line 27. Do you have something else in mind?

Comment (Pscheidl) [reply h2o-core/src/test/java/water/rapids/ast/prims/misc/SetClusterPropertyTaskTest.java]: No, I originally added a second check after the second rapids call, but that is indeed in vain.

**Pull request merged**